ignition/modules/services/ServicesModule.ts

```diff
@@ -1,8 +1,13 @@
 import { buildModule } from '@nomicfoundation/hardhat-ignition/modules';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I'd prefer to just globally fix this by adding `"ignition/**/*.ts"` to line 53 of .eslintrc.json